cypress/e2e/parallel-4/deliveryAddress.cy.ts

```diff
@@ -76,29 +77,38 @@ describe('Delivery address', () => {
 		cy.get('@address_update.all').should('have.length', 1);
 	});
 
-	it('Shows updated address when returning to manage subscription page', () => {
+	it('Cannot update National delivery address. Navigating from account overview', () => {
 		cy.intercept('GET', '/api/me/mma', {
 			statusCode: 200,
-			body: toMembersDataApiResponse(
-				guardianWeeklyPaidByCard(),
-				supporterPlus(),
-			),
+			body: toMembersDataApiResponse(nationalDelivery(), supporterPlus()),
 		}).as('mma');
 
-		cy.intercept('GET', '/mpapi/user/mobile-subscriptions', {
-			statusCode: 200,
-			body: { subscriptions: [] },
-		}).as('mobile_subscriptions');
+		cy.visit('/');
+		cy.wait('@mma');
+		cy.wait('@cancelled');
+		cy.wait('@mobile_subscriptions');
+		cy.wait('@single_contributions');
 
-		cy.intercept('GET', '/api/me/one-off-contributions', {
+		cy.findByText('Manage subscription').click();
+
+		cy.intercept('GET', '/api/me/mma?productType=ContentSubscription', {
 			statusCode: 200,
-			body: [],
-		}).as('single_contributions');
+			body: toMembersDataApiResponse(nationalDelivery(), supporterPlus()),
+		});
 
-		cy.intercept('PUT', '/api/delivery/address/update/**', {
+		cy.findByText('Manage delivery address').click();
+
+		cy.findByText(/Changed address?/).should('exist');
+	});
+
+	it('Shows updated address when returning to manage subscription page', () => {
+		cy.intercept('GET', '/api/me/mma', {
 			statusCode: 200,
-			body: 'success',
-		}).as('address_update');
+			body: toMembersDataApiResponse(
+				guardianWeeklyPaidByCard(),
+				supporterPlus(),
+			),
+		}).as('mma');
 
```

Comment:
Is it worth adding in a test to check the edge case where someone has a home delivery and a national delivery they cannot update their address?

Reply:
yeah that's a good shout

Reply:
will do in separate PR